### PR TITLE
Enable creation bytecode option

### PR DESCRIPTION
**Pull request by cryptobarbossa**
Enable runtime and creation bytecode options to be displayed with the appropriate header.

### CODE REVIEW

**Comment by JoranHonig** [comment]
@cryptobarbossa Awesome thanks! But there is a case where both runtime and creation bytecode are available (if a solidity contract is provided). And I think that this pr doesn't really account for that,

Could you change the logic to something that checks whether the bytecode is available vs whether the  argument `args.code` is used?

**Comment by cryptobarbossa** [comment]
@JoranHonig Thanks for highlighting that. Added.

File: mythril/interfaces/cli.py [hunk 1]
```diff
@@ -364,10 +365,16 @@ def main():
...
-                0
-            ].get_easm()  # or mythril.disassemble(mythril.contracts[0])
-            sys.stdout.write(easm_text)
+            output = ""
+            easm_text = mythril.contracts[0].get_easm()
+            # or mythril.disassemble(mythril.contracts[0])
+
+            if easm_text:
+                output += "Runtime Disassembly: \n" + easm_text
+            if code:
```
Comment (JoranHonig) [inline mythril/interfaces/cli.py]:
```suggestion
            if mythril.contracts[0].code:
```

File: mythril/interfaces/cli.py [hunk 2]
```diff
@@ -364,10 +365,16 @@ def main():
...
         elif args.disassemble:
-            easm_text = mythril.contracts[
-                0
-            ].get_easm()  # or mythril.disassemble(mythril.contracts[0])
-            sys.stdout.write(easm_text)
+            output = ""
+            easm_text = mythril.contracts[0].get_easm()
+            # or mythril.disassemble(mythril.contracts[0])
+
+            if easm_text:
```
Comment (JoranHonig) [inline mythril/interfaces/cli.py]:
```suggestion
            if mythril.contracts[0].code::
```

Comment (JoranHonig) [reply mythril/interfaces/cli.py]: And then you can move `easm_text = mythril.contracts[0].get_easm()` to this if statement

File: mythril/interfaces/cli.py [hunk 3]
```diff
@@ -364,10 +365,16 @@ def main():
...
+                output += "Runtime Disassembly: \n" + easm_text
+            if code:
+                output += "Disassembly: \n" + mythril.contracts[0].get_creation_easm()
+
+            sys.stdout.write(output)
```
Comment (JoranHonig) [inline mythril/interfaces/cli.py]: Why not print?

**Comment by cryptobarbossa** [comment]
Neat solutions @JoranHonig

File: mythril/interfaces/cli.py (hunk 3 above)
Comment (nbanmp) [reply mythril/interfaces/cli.py]: I think you want keep `sys.stdout.write` here. The `print()` adds an extra newline.

**Pull request merged**